bitnami/metrics-server/Chart.yaml

```diff
@@ -23,4 +23,5 @@ name: metrics-server
 sources:
   - https://github.com/bitnami/bitnami-docker-metrics-server
   - https://github.com/kubernetes-incubator/metrics-server
-version: 5.8.13
+version: 5.8.14
+
```

Comment:
Could you remove the black line?

Reply:
Removed. That's what I get for resolving conflicts through the web interface 😔